runtime/gc_vlhgc/CopyScanCacheChunkVLHGCInHeap.hpp

```diff
@@ -46,15 +46,15 @@ class MM_CopyScanCacheChunkVLHGCInHeap : public MM_CopyScanCacheChunkVLHGC
 	 * @param env[in] A GC thread
 	 * @return The number of caches which will be allocated as part of an instance of the receiver
 	 */
-	static UDATA numberOfCachesInChunk(MM_EnvironmentVLHGC *env);
+	static uintptr_t numberOfCachesInChunk(MM_EnvironmentVLHGC *env);
```

Comment:
The corresponding `.cpp` file should use the same type names.

Reply:
If I recall correctly, the `.cpp` file was updated properly in an earlier version of this PR; I'm not sure why that got reverted.